addons/binding/org.openhab.binding.regoheatpump/README.md

```diff
@@ -0,0 +1,222 @@
+| deviceValues#additionalHeat6kW    | Switch
+| deviceValues#radiatorPump         | Switch
+| deviceValues#heatFluidPump        | Switch
+| deviceValues#switchValue          | Switch
+| deviceValues#alarm                | Switch
+| settings#heatCurve                | Number
+| settings#heatCurveFineAdj         | Temperature
+| registersettings#indoorTempSetting | Temperature
+| settings#curveInflByInTemp         | Number
+| settings#adjCurveAt20              | Temperature
+| settings#adjCurveAt15              | Temperature
+| settings#adjCurveAt10              | Temperature
+| settings#adjCurveAt5               | Temperature
+| settings#adjCurveAt0               | Temperature
+| settings#adjCurveAtMinus5          | Temperature
+| settings#adjCurveAtMinus10         | Temperature
+| settings#adjCurveAtMinus15         | Temperature
+| settings#adjCurveAtMinus20         | Temperature
+| settings#adjCurveAtMinus25         | Temperature
+| settings#adjCurveAtMinus30         | Temperature
+| settings#adjCurveAtMinus35         | Temperature
+| settings#heatCurveCouplingDiff     | Temperature
+
+
+## The Husdata interface
+
+The [Husdata](http://www.husdata.se/) interface bridges the often complex communication methods with a heat pump controller and provides access through a simple standard interface over RS-232.
+
+Supported heat pump models
+
+| Heat pump models | Technical |
+| ------------------------ |-------------------|
+| IVT Greenline / Optima 900 | Rego 600 Serial
+| IVT 490 | Rego 400 Serial
+| IVT Premiumline X, Optima/290-AW | Rego 800, Can bus
+| IVT Greenline HE/HC/HA+Prem HQ/EQ | Rego 1000, Can bus
+| NIBE xx45 | EB100, RS-485
+| NIBE Fighter series  | Styr 2002, RS-485
+| Thermia Diplomat series | 901510, i2c
+
+Above list is informational, please consult with the Husdata interface provider for further details.
+
+### Thing configuration
+
+Two connection types are supported:
+* TCP/IP and
+* serial (RS232).
+
+#### TCP/IP connection
+
+A transparent bridge between the Husdata interface and network (i.e. wifi) is used.
+
+There are many existing project providing such functionality, i.e. [ser2net](http://ser2net.sourceforge.net/).
+
+Configuration of the TCP/IP thing:
+
+ - address: the hostname/IP address of the transparent bridge on the local network - mandatory,
+ - tcpPort: the port number to use to connect to the transparent bridge - optional, defaults to 9265.
+
+Example thing definition:
+ ```
+ regoheatpump:ipHusdata:ivtIP [ address="192.168.2.50", tcpPort="9265" ]
+ ```
+
+#### Serial connection
+
+One can connect the Husdata interface directly to a computer running the OH.
+
+Parameters:
+
+ - portName: the name of the serial port on your computer - mandatory.
+
+Example thing definition:
+
+```
+regoheatpump:serialHusdata:ivtSerial [ portName="COM3" ]
+```
+
+
+### Channels
+
+Below is the list of supported channels, all values are read only:
+
+| H1 ID | Name        | Channel Type ID | Item Type           
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍